[2023-02-17T11:49:51.454+0000] {processor.py:153} INFO - Started process (PID=111) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T11:49:51.457+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T11:49:51.482+0000] {logging_mixin.py:137} INFO - [2023-02-17T11:49:51.459+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T11:49:52.068+0000] {logging_mixin.py:137} INFO - [2023-02-17T11:49:52.022+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T11:49:52.069+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T11:49:52.484+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 1.082 seconds
[2023-02-17T11:51:09.191+0000] {processor.py:153} INFO - Started process (PID=185) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T11:51:09.206+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T11:51:09.209+0000] {logging_mixin.py:137} INFO - [2023-02-17T11:51:09.208+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T11:51:09.351+0000] {logging_mixin.py:137} INFO - [2023-02-17T11:51:09.348+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T11:51:09.364+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T11:51:09.661+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 0.492 seconds
[2023-02-17T11:51:42.913+0000] {processor.py:153} INFO - Started process (PID=240) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T11:51:42.916+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T11:51:43.010+0000] {logging_mixin.py:137} INFO - [2023-02-17T11:51:42.917+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T11:51:43.368+0000] {logging_mixin.py:137} INFO - [2023-02-17T11:51:43.234+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T11:51:43.370+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T11:51:43.782+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 1.049 seconds
[2023-02-17T11:52:14.739+0000] {processor.py:153} INFO - Started process (PID=290) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T11:52:14.804+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T11:52:14.830+0000] {logging_mixin.py:137} INFO - [2023-02-17T11:52:14.817+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T11:52:15.038+0000] {logging_mixin.py:137} INFO - [2023-02-17T11:52:14.958+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T11:52:15.039+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T11:52:15.428+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 0.707 seconds
[2023-02-17T11:52:51.773+0000] {processor.py:153} INFO - Started process (PID=342) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T11:52:51.778+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T11:52:51.780+0000] {logging_mixin.py:137} INFO - [2023-02-17T11:52:51.779+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T11:52:51.829+0000] {logging_mixin.py:137} INFO - [2023-02-17T11:52:51.826+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T11:52:51.830+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T11:52:52.009+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 0.257 seconds
[2023-02-17T11:53:24.012+0000] {processor.py:153} INFO - Started process (PID=398) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T11:53:24.040+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T11:53:24.091+0000] {logging_mixin.py:137} INFO - [2023-02-17T11:53:24.091+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T11:53:24.192+0000] {logging_mixin.py:137} INFO - [2023-02-17T11:53:24.188+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T11:53:24.220+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T11:53:24.644+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 0.655 seconds
[2023-02-17T11:53:55.944+0000] {processor.py:153} INFO - Started process (PID=457) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T11:53:55.968+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T11:53:55.970+0000] {logging_mixin.py:137} INFO - [2023-02-17T11:53:55.969+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T11:53:56.003+0000] {logging_mixin.py:137} INFO - [2023-02-17T11:53:55.996+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T11:53:56.005+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T11:53:56.162+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 0.230 seconds
[2023-02-17T11:54:27.777+0000] {processor.py:153} INFO - Started process (PID=508) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T11:54:27.779+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T11:54:27.781+0000] {logging_mixin.py:137} INFO - [2023-02-17T11:54:27.780+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T11:54:27.873+0000] {logging_mixin.py:137} INFO - [2023-02-17T11:54:27.859+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T11:54:27.874+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T11:54:28.105+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 0.432 seconds
[2023-02-17T11:55:01.006+0000] {processor.py:153} INFO - Started process (PID=566) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T11:55:01.021+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T11:55:01.024+0000] {logging_mixin.py:137} INFO - [2023-02-17T11:55:01.024+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T11:55:01.072+0000] {logging_mixin.py:137} INFO - [2023-02-17T11:55:01.070+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T11:55:01.080+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T11:55:01.384+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 0.420 seconds
[2023-02-17T11:55:32.323+0000] {processor.py:153} INFO - Started process (PID=617) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T11:55:32.338+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T11:55:32.353+0000] {logging_mixin.py:137} INFO - [2023-02-17T11:55:32.340+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T11:55:32.433+0000] {logging_mixin.py:137} INFO - [2023-02-17T11:55:32.424+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T11:55:32.440+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T11:55:32.662+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 0.361 seconds
[2023-02-17T11:56:05.297+0000] {processor.py:153} INFO - Started process (PID=676) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T11:56:05.319+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T11:56:05.368+0000] {logging_mixin.py:137} INFO - [2023-02-17T11:56:05.367+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T11:56:05.428+0000] {logging_mixin.py:137} INFO - [2023-02-17T11:56:05.426+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T11:56:05.429+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T11:56:05.780+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 0.537 seconds
[2023-02-17T11:56:36.890+0000] {processor.py:153} INFO - Started process (PID=728) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T11:56:36.896+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T11:56:36.899+0000] {logging_mixin.py:137} INFO - [2023-02-17T11:56:36.898+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T11:56:36.929+0000] {logging_mixin.py:137} INFO - [2023-02-17T11:56:36.926+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T11:56:36.930+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T11:56:37.054+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 0.176 seconds
[2023-02-17T11:57:08.887+0000] {processor.py:153} INFO - Started process (PID=782) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T11:57:08.889+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T11:57:08.904+0000] {logging_mixin.py:137} INFO - [2023-02-17T11:57:08.903+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T11:57:08.981+0000] {logging_mixin.py:137} INFO - [2023-02-17T11:57:08.975+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T11:57:08.984+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T11:57:09.225+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 0.385 seconds
[2023-02-17T11:57:40.095+0000] {processor.py:153} INFO - Started process (PID=836) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T11:57:40.099+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T11:57:40.102+0000] {logging_mixin.py:137} INFO - [2023-02-17T11:57:40.101+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T11:57:40.153+0000] {logging_mixin.py:137} INFO - [2023-02-17T11:57:40.138+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T11:57:40.154+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T11:57:40.285+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 0.203 seconds
[2023-02-17T11:58:11.355+0000] {processor.py:153} INFO - Started process (PID=895) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T11:58:11.361+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T11:58:11.379+0000] {logging_mixin.py:137} INFO - [2023-02-17T11:58:11.378+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T11:58:11.553+0000] {logging_mixin.py:137} INFO - [2023-02-17T11:58:11.522+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T11:58:11.555+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T11:58:11.813+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 0.576 seconds
[2023-02-17T11:58:42.014+0000] {processor.py:153} INFO - Started process (PID=947) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T11:58:42.021+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T11:58:42.023+0000] {logging_mixin.py:137} INFO - [2023-02-17T11:58:42.023+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T11:58:42.054+0000] {logging_mixin.py:137} INFO - [2023-02-17T11:58:42.051+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T11:58:42.055+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T11:58:42.156+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 0.158 seconds
[2023-02-17T11:59:16.803+0000] {processor.py:153} INFO - Started process (PID=1004) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T11:59:16.816+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T11:59:16.818+0000] {logging_mixin.py:137} INFO - [2023-02-17T11:59:16.817+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T11:59:16.846+0000] {logging_mixin.py:137} INFO - [2023-02-17T11:59:16.843+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T11:59:16.847+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T11:59:16.950+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 0.164 seconds
[2023-02-17T11:59:47.243+0000] {processor.py:153} INFO - Started process (PID=1063) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T11:59:47.245+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T11:59:47.247+0000] {logging_mixin.py:137} INFO - [2023-02-17T11:59:47.246+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T11:59:47.279+0000] {logging_mixin.py:137} INFO - [2023-02-17T11:59:47.277+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T11:59:47.280+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T11:59:47.649+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 0.419 seconds
[2023-02-17T12:00:20.956+0000] {processor.py:153} INFO - Started process (PID=1112) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T12:00:20.960+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T12:00:20.967+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:00:20.962+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:00:21.048+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:00:21.046+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T12:00:21.049+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:00:21.286+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 0.345 seconds
[2023-02-17T12:01:25.254+0000] {processor.py:153} INFO - Started process (PID=1170) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T12:01:25.318+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T12:01:25.373+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:01:25.320+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:01:25.756+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:01:25.753+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T12:01:25.762+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:01:26.104+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 0.911 seconds
[2023-02-17T12:02:16.141+0000] {processor.py:153} INFO - Started process (PID=1230) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T12:02:16.171+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T12:02:16.174+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:02:16.173+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:02:16.411+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:02:16.407+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T12:02:16.480+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:02:17.025+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 0.903 seconds
[2023-02-17T12:02:50.729+0000] {processor.py:153} INFO - Started process (PID=1278) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T12:02:50.757+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T12:02:50.764+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:02:50.763+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:02:50.910+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:02:50.896+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T12:02:50.912+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:02:51.171+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 0.477 seconds
[2023-02-17T12:03:24.171+0000] {processor.py:153} INFO - Started process (PID=1316) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T12:03:24.178+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T12:03:24.181+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:03:24.180+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:03:24.270+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:03:24.267+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T12:03:24.288+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:03:24.447+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 0.304 seconds
[2023-02-17T12:04:13.203+0000] {processor.py:153} INFO - Started process (PID=1377) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T12:04:13.219+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T12:04:13.252+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:04:13.252+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:04:13.412+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:04:13.397+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T12:04:13.427+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:04:14.620+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 1.560 seconds
[2023-02-17T12:05:05.492+0000] {processor.py:153} INFO - Started process (PID=1434) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T12:05:05.536+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T12:05:05.590+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:05:05.590+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:05:05.776+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:05:05.762+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T12:05:05.778+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:05:06.166+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 0.888 seconds
[2023-02-17T12:05:48.711+0000] {processor.py:153} INFO - Started process (PID=1484) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T12:05:48.728+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T12:05:48.755+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:05:48.735+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:05:48.927+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:05:48.906+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T12:05:48.936+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:05:49.218+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 0.572 seconds
[2023-02-17T12:06:29.063+0000] {processor.py:153} INFO - Started process (PID=1532) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T12:06:29.130+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T12:06:29.170+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:06:29.133+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:06:29.412+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:06:29.389+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T12:06:29.413+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:06:29.997+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 1.066 seconds
[2023-02-17T12:07:07.232+0000] {processor.py:153} INFO - Started process (PID=1584) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T12:07:07.295+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T12:07:07.302+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:07:07.297+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:07:07.366+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:07:07.360+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T12:07:07.368+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:07:07.712+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 0.514 seconds
[2023-02-17T12:07:39.917+0000] {processor.py:153} INFO - Started process (PID=1630) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T12:07:39.927+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T12:07:39.933+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:07:39.929+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:07:40.084+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:07:40.080+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T12:07:40.085+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:07:40.543+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 0.691 seconds
[2023-02-17T12:08:12.308+0000] {processor.py:153} INFO - Started process (PID=1663) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T12:08:12.428+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T12:08:12.431+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:08:12.430+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:08:12.605+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:08:12.584+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T12:08:12.606+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:08:12.799+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 0.544 seconds
[2023-02-17T12:08:46.275+0000] {processor.py:153} INFO - Started process (PID=1712) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T12:08:46.306+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T12:08:46.308+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:08:46.308+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:08:46.596+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:08:46.593+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T12:08:46.638+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:08:47.035+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 0.875 seconds
[2023-02-17T12:09:33.546+0000] {processor.py:153} INFO - Started process (PID=1765) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T12:09:33.577+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T12:09:33.587+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:09:33.586+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:09:33.715+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:09:33.700+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T12:09:33.717+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:09:33.935+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 0.452 seconds
[2023-02-17T12:10:34.482+0000] {processor.py:153} INFO - Started process (PID=1826) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T12:10:34.512+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T12:10:34.514+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:10:34.514+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:10:34.815+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:10:34.784+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T12:10:34.817+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:10:35.271+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 0.853 seconds
[2023-02-17T12:11:06.225+0000] {processor.py:153} INFO - Started process (PID=1865) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T12:11:06.240+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T12:11:06.242+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:11:06.242+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:11:06.440+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:11:06.436+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T12:11:06.442+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:11:06.984+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 0.852 seconds
[2023-02-17T12:11:53.310+0000] {processor.py:153} INFO - Started process (PID=1903) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T12:11:53.327+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T12:11:53.342+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:11:53.329+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:11:53.616+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:11:53.576+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T12:11:53.617+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:11:54.007+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 0.820 seconds
[2023-02-17T12:12:53.982+0000] {processor.py:153} INFO - Started process (PID=1954) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T12:12:54.027+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T12:12:54.029+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:12:54.029+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:12:54.157+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:12:54.140+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T12:12:54.171+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:12:54.487+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 0.555 seconds
[2023-02-17T12:14:14.610+0000] {processor.py:153} INFO - Started process (PID=2012) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T12:14:14.643+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T12:14:14.645+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:14:14.645+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:14:15.043+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:14:14.931+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T12:14:15.045+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:14:16.059+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 1.496 seconds
[2023-02-17T12:16:19.722+0000] {processor.py:153} INFO - Started process (PID=2102) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T12:16:19.803+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T12:16:19.817+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:16:19.817+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:16:19.921+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:16:19.903+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T12:16:19.937+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:16:20.464+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 1.015 seconds
[2023-02-17T12:18:13.511+0000] {processor.py:153} INFO - Started process (PID=2182) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T12:18:13.542+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T12:18:13.544+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:18:13.544+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:18:14.368+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:18:14.353+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T12:18:14.401+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:18:20.326+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 7.511 seconds
[2023-02-17T12:20:15.763+0000] {processor.py:153} INFO - Started process (PID=2270) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T12:20:15.780+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T12:20:15.859+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:20:15.782+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:20:16.222+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:20:16.204+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T12:20:16.235+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:20:16.780+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 1.416 seconds
[2023-02-17T12:21:53.998+0000] {processor.py:153} INFO - Started process (PID=2338) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T12:21:54.059+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T12:21:54.075+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:21:54.062+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:21:54.238+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:21:54.222+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T12:21:54.251+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:21:55.341+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 1.537 seconds
[2023-02-17T12:23:33.823+0000] {processor.py:153} INFO - Started process (PID=2410) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T12:23:33.968+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T12:23:33.983+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:23:33.970+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:23:34.424+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:23:34.387+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T12:23:34.426+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:23:35.592+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 2.042 seconds
[2023-02-17T12:24:36.674+0000] {processor.py:153} INFO - Started process (PID=2465) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T12:24:36.689+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T12:24:36.692+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:24:36.691+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:24:36.820+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:24:36.788+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T12:24:36.855+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:24:37.282+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 0.713 seconds
[2023-02-17T12:25:08.670+0000] {processor.py:153} INFO - Started process (PID=2503) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T12:25:08.685+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T12:25:08.687+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:25:08.686+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:25:08.767+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:25:08.764+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T12:25:08.768+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:25:08.972+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 0.339 seconds
[2023-02-17T12:25:53.656+0000] {processor.py:153} INFO - Started process (PID=2557) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T12:25:53.677+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T12:25:53.686+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:25:53.679+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:25:53.782+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:25:53.768+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T12:25:53.783+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:25:54.115+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 0.509 seconds
[2023-02-17T12:26:26.912+0000] {processor.py:153} INFO - Started process (PID=2601) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T12:26:26.932+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T12:26:26.937+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:26:26.937+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:26:27.037+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:26:27.020+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T12:26:27.038+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:26:27.322+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 0.453 seconds
[2023-02-17T12:27:13.921+0000] {processor.py:153} INFO - Started process (PID=2666) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T12:27:13.936+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T12:27:13.939+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:27:13.938+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:27:14.003+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:27:13.996+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T12:27:14.019+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:27:14.569+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 0.666 seconds
[2023-02-17T12:27:50.700+0000] {processor.py:153} INFO - Started process (PID=2720) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T12:27:50.716+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T12:27:50.720+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:27:50.719+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:27:50.785+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:27:50.782+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T12:27:50.799+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:27:51.113+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 0.438 seconds
[2023-02-17T12:28:23.935+0000] {processor.py:153} INFO - Started process (PID=2764) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T12:28:24.020+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T12:28:24.022+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:28:24.022+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:28:24.082+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:28:24.080+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T12:28:24.095+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:28:24.330+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 0.419 seconds
[2023-02-17T12:28:57.560+0000] {processor.py:153} INFO - Started process (PID=2820) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T12:28:57.573+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T12:28:57.575+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:28:57.574+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:28:57.642+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:28:57.640+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T12:28:57.655+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:28:57.933+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 0.449 seconds
[2023-02-17T12:29:31.681+0000] {processor.py:153} INFO - Started process (PID=2873) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T12:29:31.801+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T12:29:31.803+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:29:31.802+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:29:32.060+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:29:32.014+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T12:29:32.061+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:29:32.598+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 0.970 seconds
[2023-02-17T12:30:04.952+0000] {processor.py:153} INFO - Started process (PID=2918) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T12:30:04.965+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T12:30:04.968+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:30:04.967+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:30:05.137+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:30:05.120+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T12:30:05.165+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:30:05.782+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 0.863 seconds
[2023-02-17T12:30:43.234+0000] {processor.py:153} INFO - Started process (PID=2978) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T12:30:43.266+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T12:30:43.280+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:30:43.280+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:30:43.437+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:30:43.432+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T12:30:43.438+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:30:43.820+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 0.614 seconds
[2023-02-17T12:31:16.406+0000] {processor.py:153} INFO - Started process (PID=3034) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T12:31:16.548+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T12:31:16.550+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:31:16.549+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:31:16.970+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:31:16.940+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T12:31:16.971+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:31:17.433+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 1.154 seconds
[2023-02-17T12:31:49.647+0000] {processor.py:153} INFO - Started process (PID=3071) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T12:31:49.702+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T12:31:49.715+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:31:49.714+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:31:49.871+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:31:49.856+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T12:31:49.872+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:31:50.146+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 0.527 seconds
[2023-02-17T12:32:23.996+0000] {processor.py:153} INFO - Started process (PID=3116) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T12:32:24.002+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T12:32:24.004+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:32:24.003+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:32:24.037+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:32:24.028+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T12:32:24.039+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:32:24.238+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 0.259 seconds
[2023-02-17T12:32:55.016+0000] {processor.py:153} INFO - Started process (PID=3175) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T12:32:55.026+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T12:32:55.032+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:32:55.032+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:32:55.152+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:32:55.122+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T12:32:55.153+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:32:55.412+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 0.458 seconds
[2023-02-17T12:33:25.870+0000] {processor.py:153} INFO - Started process (PID=3228) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T12:33:25.898+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T12:33:25.911+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:33:25.911+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:33:26.061+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:33:26.046+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T12:33:26.063+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:33:26.206+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 0.367 seconds
[2023-02-17T12:33:56.640+0000] {processor.py:153} INFO - Started process (PID=3281) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T12:33:56.642+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T12:33:56.664+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:33:56.644+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:33:56.763+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:33:56.761+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T12:33:56.764+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:33:56.943+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 0.355 seconds
[2023-02-17T12:34:30.665+0000] {processor.py:153} INFO - Started process (PID=3336) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T12:34:30.689+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T12:34:30.691+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:34:30.690+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:34:30.761+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:34:30.758+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T12:34:30.771+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:34:30.976+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 0.330 seconds
[2023-02-17T12:35:11.156+0000] {processor.py:153} INFO - Started process (PID=3392) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T12:35:11.233+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T12:35:11.235+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:35:11.235+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:35:11.424+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:35:11.395+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T12:35:11.425+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:35:11.968+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 0.662 seconds
[2023-02-17T12:35:47.195+0000] {processor.py:153} INFO - Started process (PID=3440) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T12:35:47.213+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T12:35:47.215+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:35:47.214+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:35:47.618+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:35:47.492+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T12:35:47.619+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:35:47.943+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 0.830 seconds
[2023-02-17T12:36:22.112+0000] {processor.py:153} INFO - Started process (PID=3484) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T12:36:22.143+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T12:36:22.145+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:36:22.145+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:36:22.212+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:36:22.209+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T12:36:22.213+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:36:22.662+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 0.639 seconds
[2023-02-17T12:36:54.672+0000] {processor.py:153} INFO - Started process (PID=3535) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T12:36:54.675+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T12:36:54.705+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:36:54.704+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:36:54.831+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:36:54.803+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T12:36:54.832+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:36:55.073+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 0.484 seconds
[2023-02-17T12:37:34.843+0000] {processor.py:153} INFO - Started process (PID=3585) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T12:37:34.864+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T12:37:34.867+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:37:34.866+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:37:35.081+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:37:35.066+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T12:37:35.082+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:37:35.644+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 0.913 seconds
[2023-02-17T12:38:21.299+0000] {processor.py:153} INFO - Started process (PID=3644) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T12:38:21.325+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T12:38:21.328+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:38:21.328+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:38:21.473+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:38:21.471+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T12:38:21.474+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:38:21.729+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 0.486 seconds
[2023-02-17T12:39:00.395+0000] {processor.py:153} INFO - Started process (PID=3701) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T12:39:00.419+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T12:39:00.421+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:39:00.421+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:39:00.522+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:39:00.519+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T12:39:00.527+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:39:00.897+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 0.552 seconds
[2023-02-17T12:39:31.825+0000] {processor.py:153} INFO - Started process (PID=3756) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T12:39:31.832+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T12:39:31.835+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:39:31.834+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:39:31.934+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:39:31.920+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T12:39:31.935+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:39:32.237+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 0.439 seconds
[2023-02-17T12:40:03.488+0000] {processor.py:153} INFO - Started process (PID=3803) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T12:40:03.492+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T12:40:03.501+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:40:03.500+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:40:03.613+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:40:03.608+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T12:40:03.614+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:40:03.753+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 0.276 seconds
[2023-02-17T12:40:34.204+0000] {processor.py:153} INFO - Started process (PID=3856) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T12:40:34.219+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T12:40:34.221+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:40:34.221+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:40:34.384+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:40:34.344+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T12:40:34.385+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:40:34.746+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 0.588 seconds
[2023-02-17T12:41:07.524+0000] {processor.py:153} INFO - Started process (PID=3897) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T12:41:07.589+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T12:41:07.620+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:41:07.619+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:41:07.748+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:41:07.734+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T12:41:07.761+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:41:08.275+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 0.815 seconds
[2023-02-17T12:42:39.568+0000] {processor.py:153} INFO - Started process (PID=3975) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T12:42:39.583+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T12:42:39.586+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:42:39.585+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:42:39.841+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:42:39.810+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T12:42:39.870+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:42:40.633+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 1.145 seconds
[2023-02-17T12:43:33.885+0000] {processor.py:153} INFO - Started process (PID=4029) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T12:43:33.930+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T12:43:33.933+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:43:33.933+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:43:34.196+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:43:34.115+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T12:43:34.242+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:43:34.728+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 0.878 seconds
[2023-02-17T12:44:12.608+0000] {processor.py:153} INFO - Started process (PID=4059) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T12:44:12.704+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T12:44:12.706+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:44:12.706+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:44:12.974+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:44:12.944+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T12:44:12.976+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:44:13.439+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 0.961 seconds
[2023-02-17T12:45:12.217+0000] {processor.py:153} INFO - Started process (PID=4111) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T12:45:12.293+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T12:45:12.295+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:45:12.295+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:45:12.837+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:45:12.696+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T12:45:12.839+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:45:13.982+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 1.992 seconds
[2023-02-17T12:46:05.200+0000] {processor.py:153} INFO - Started process (PID=4164) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T12:46:05.222+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T12:46:05.233+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:46:05.232+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:46:05.330+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:46:05.327+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T12:46:05.331+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:46:05.517+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 0.404 seconds
[2023-02-17T12:47:09.796+0000] {processor.py:153} INFO - Started process (PID=4232) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T12:47:09.865+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T12:47:09.869+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:47:09.868+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:47:09.971+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:47:09.956+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T12:47:09.985+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:47:10.314+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 0.613 seconds
[2023-02-17T12:48:09.238+0000] {processor.py:153} INFO - Started process (PID=4295) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T12:48:09.243+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T12:48:09.257+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:48:09.256+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:48:09.457+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:48:09.454+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T12:48:09.470+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:48:09.887+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 0.728 seconds
[2023-02-17T12:49:05.102+0000] {processor.py:153} INFO - Started process (PID=4357) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T12:49:05.214+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T12:49:05.217+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:49:05.217+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:49:05.407+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:49:05.375+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T12:49:05.408+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:49:06.031+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 1.009 seconds
[2023-02-17T12:49:43.948+0000] {processor.py:153} INFO - Started process (PID=4392) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T12:49:43.972+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T12:49:43.975+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:49:43.974+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:49:44.416+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:49:44.398+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T12:49:44.477+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:49:45.981+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 2.199 seconds
[2023-02-17T12:50:48.310+0000] {processor.py:153} INFO - Started process (PID=4444) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T12:50:48.340+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T12:50:48.372+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:50:48.371+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:50:48.677+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:50:48.629+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T12:50:48.747+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:50:49.708+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 1.839 seconds
[2023-02-17T12:51:29.991+0000] {processor.py:153} INFO - Started process (PID=4491) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T12:51:30.071+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T12:51:30.118+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:51:30.117+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:51:30.239+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:51:30.224+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T12:51:30.240+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:51:30.706+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 0.827 seconds
[2023-02-17T12:52:14.914+0000] {processor.py:153} INFO - Started process (PID=4544) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T12:52:14.961+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T12:52:14.975+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:52:14.975+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:52:15.159+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:52:15.128+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T12:52:15.161+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:52:15.975+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 1.128 seconds
[2023-02-17T12:53:17.116+0000] {processor.py:153} INFO - Started process (PID=4610) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T12:53:17.136+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T12:53:17.139+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:53:17.138+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:53:17.304+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:53:17.300+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T12:53:17.339+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:53:17.859+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 0.817 seconds
[2023-02-17T12:54:35.579+0000] {processor.py:153} INFO - Started process (PID=4678) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T12:54:35.610+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T12:54:35.613+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:54:35.612+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:54:36.011+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:54:36.006+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T12:54:36.013+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:54:36.386+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 0.872 seconds
[2023-02-17T12:55:50.982+0000] {processor.py:153} INFO - Started process (PID=4738) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T12:55:51.012+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T12:55:51.015+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:55:51.014+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:55:51.677+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:55:51.646+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T12:55:51.679+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:55:52.740+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 1.989 seconds
[2023-02-17T12:57:07.063+0000] {processor.py:153} INFO - Started process (PID=4796) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T12:57:07.107+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T12:57:07.111+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:57:07.110+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:57:07.267+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:57:07.252+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T12:57:07.269+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:57:07.917+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 0.949 seconds
[2023-02-17T12:58:27.370+0000] {processor.py:153} INFO - Started process (PID=4860) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T12:58:27.423+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T12:58:27.426+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:58:27.425+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:58:27.714+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:58:27.698+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T12:58:27.743+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:58:28.448+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 1.175 seconds
[2023-02-17T12:59:42.722+0000] {processor.py:153} INFO - Started process (PID=4921) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T12:59:42.738+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T12:59:42.810+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:59:42.810+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:59:43.323+0000] {logging_mixin.py:137} INFO - [2023-02-17T12:59:43.283+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T12:59:43.324+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T12:59:43.716+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 1.255 seconds
[2023-02-17T13:00:18.753+0000] {processor.py:153} INFO - Started process (PID=4966) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T13:00:18.905+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T13:00:18.908+0000] {logging_mixin.py:137} INFO - [2023-02-17T13:00:18.907+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T13:00:19.069+0000] {logging_mixin.py:137} INFO - [2023-02-17T13:00:19.065+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T13:00:19.071+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T13:00:19.440+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 0.798 seconds
[2023-02-17T13:01:15.456+0000] {processor.py:153} INFO - Started process (PID=5023) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T13:01:15.518+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T13:01:15.521+0000] {logging_mixin.py:137} INFO - [2023-02-17T13:01:15.521+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T13:01:15.697+0000] {logging_mixin.py:137} INFO - [2023-02-17T13:01:15.681+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T13:01:15.727+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T13:01:16.680+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 1.337 seconds
[2023-02-17T13:02:03.740+0000] {processor.py:153} INFO - Started process (PID=5065) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T13:02:03.834+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T13:02:03.864+0000] {logging_mixin.py:137} INFO - [2023-02-17T13:02:03.837+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T13:02:04.457+0000] {logging_mixin.py:137} INFO - [2023-02-17T13:02:04.424+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T13:02:04.459+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T13:02:05.710+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 2.110 seconds
[2023-02-17T13:03:03.786+0000] {processor.py:153} INFO - Started process (PID=5116) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T13:03:03.802+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T13:03:03.820+0000] {logging_mixin.py:137} INFO - [2023-02-17T13:03:03.819+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T13:03:04.011+0000] {logging_mixin.py:137} INFO - [2023-02-17T13:03:03.996+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T13:03:04.012+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T13:03:05.682+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 2.389 seconds
[2023-02-17T13:03:50.893+0000] {processor.py:153} INFO - Started process (PID=5162) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T13:03:50.905+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T13:03:50.908+0000] {logging_mixin.py:137} INFO - [2023-02-17T13:03:50.908+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T13:03:50.975+0000] {logging_mixin.py:137} INFO - [2023-02-17T13:03:50.972+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T13:03:50.977+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T13:03:51.235+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 0.358 seconds
[2023-02-17T13:04:21.890+0000] {processor.py:153} INFO - Started process (PID=5209) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T13:04:21.919+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T13:04:21.922+0000] {logging_mixin.py:137} INFO - [2023-02-17T13:04:21.921+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T13:04:22.108+0000] {logging_mixin.py:137} INFO - [2023-02-17T13:04:22.105+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T13:04:22.109+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T13:04:22.608+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 0.730 seconds
[2023-02-17T13:05:11.384+0000] {processor.py:153} INFO - Started process (PID=5254) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T13:05:11.400+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T13:05:11.403+0000] {logging_mixin.py:137} INFO - [2023-02-17T13:05:11.402+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T13:05:11.617+0000] {logging_mixin.py:137} INFO - [2023-02-17T13:05:11.602+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T13:05:11.618+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T13:05:11.952+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 0.814 seconds
[2023-02-17T13:05:50.393+0000] {processor.py:153} INFO - Started process (PID=5296) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T13:05:50.427+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T13:05:50.430+0000] {logging_mixin.py:137} INFO - [2023-02-17T13:05:50.430+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T13:05:50.516+0000] {logging_mixin.py:137} INFO - [2023-02-17T13:05:50.509+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T13:05:50.517+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T13:05:50.745+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 0.405 seconds
[2023-02-17T13:06:31.025+0000] {processor.py:153} INFO - Started process (PID=5352) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T13:06:31.073+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T13:06:31.076+0000] {logging_mixin.py:137} INFO - [2023-02-17T13:06:31.075+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T13:06:31.150+0000] {logging_mixin.py:137} INFO - [2023-02-17T13:06:31.145+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T13:06:31.161+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T13:06:31.351+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 0.395 seconds
[2023-02-17T13:07:25.083+0000] {processor.py:153} INFO - Started process (PID=5406) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T13:07:25.131+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T13:07:25.142+0000] {logging_mixin.py:137} INFO - [2023-02-17T13:07:25.141+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T13:07:25.806+0000] {logging_mixin.py:137} INFO - [2023-02-17T13:07:25.803+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T13:07:25.851+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T13:07:26.436+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 1.521 seconds
[2023-02-17T13:09:01.290+0000] {processor.py:153} INFO - Started process (PID=5469) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T13:09:01.340+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T13:09:01.410+0000] {logging_mixin.py:137} INFO - [2023-02-17T13:09:01.410+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T13:09:02.173+0000] {logging_mixin.py:137} INFO - [2023-02-17T13:09:02.070+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T13:09:02.174+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T13:09:02.997+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 1.897 seconds
[2023-02-17T13:09:59.771+0000] {processor.py:153} INFO - Started process (PID=5527) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T13:09:59.790+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T13:09:59.793+0000] {logging_mixin.py:137} INFO - [2023-02-17T13:09:59.792+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T13:09:59.866+0000] {logging_mixin.py:137} INFO - [2023-02-17T13:09:59.863+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T13:09:59.879+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T13:10:00.095+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 0.376 seconds
[2023-02-17T13:10:32.135+0000] {processor.py:153} INFO - Started process (PID=5582) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T13:10:32.138+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T13:10:32.145+0000] {logging_mixin.py:137} INFO - [2023-02-17T13:10:32.144+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T13:10:32.216+0000] {logging_mixin.py:137} INFO - [2023-02-17T13:10:32.203+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T13:10:32.217+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T13:10:32.601+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 0.519 seconds
[2023-02-17T13:11:10.132+0000] {processor.py:153} INFO - Started process (PID=5635) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T13:11:10.148+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T13:11:10.150+0000] {logging_mixin.py:137} INFO - [2023-02-17T13:11:10.149+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T13:11:10.355+0000] {logging_mixin.py:137} INFO - [2023-02-17T13:11:10.340+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T13:11:10.356+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T13:11:10.864+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 0.766 seconds
[2023-02-17T13:11:41.552+0000] {processor.py:153} INFO - Started process (PID=5685) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T13:11:41.565+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T13:11:41.569+0000] {logging_mixin.py:137} INFO - [2023-02-17T13:11:41.568+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T13:11:41.682+0000] {logging_mixin.py:137} INFO - [2023-02-17T13:11:41.668+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T13:11:41.683+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T13:11:42.071+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 0.566 seconds
[2023-02-17T13:12:19.969+0000] {processor.py:153} INFO - Started process (PID=5736) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T13:12:19.991+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T13:12:19.995+0000] {logging_mixin.py:137} INFO - [2023-02-17T13:12:19.994+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T13:12:20.077+0000] {logging_mixin.py:137} INFO - [2023-02-17T13:12:20.064+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T13:12:20.079+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T13:12:20.270+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 0.348 seconds
[2023-02-17T13:12:52.663+0000] {processor.py:153} INFO - Started process (PID=5787) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T13:12:52.681+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T13:12:52.683+0000] {logging_mixin.py:137} INFO - [2023-02-17T13:12:52.682+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T13:12:52.854+0000] {logging_mixin.py:137} INFO - [2023-02-17T13:12:52.839+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T13:12:52.855+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T13:12:53.120+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 0.537 seconds
[2023-02-17T13:13:30.477+0000] {processor.py:153} INFO - Started process (PID=5837) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T13:13:30.540+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T13:13:30.542+0000] {logging_mixin.py:137} INFO - [2023-02-17T13:13:30.542+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T13:13:30.675+0000] {logging_mixin.py:137} INFO - [2023-02-17T13:13:30.660+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T13:13:30.676+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T13:13:30.897+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 0.487 seconds
[2023-02-17T13:14:08.116+0000] {processor.py:153} INFO - Started process (PID=5893) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T13:14:08.194+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T13:14:08.197+0000] {logging_mixin.py:137} INFO - [2023-02-17T13:14:08.196+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T13:14:08.291+0000] {logging_mixin.py:137} INFO - [2023-02-17T13:14:08.277+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T13:14:08.292+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T13:14:08.887+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 0.805 seconds
[2023-02-17T13:14:43.479+0000] {processor.py:153} INFO - Started process (PID=5940) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T13:14:43.496+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T13:14:43.498+0000] {logging_mixin.py:137} INFO - [2023-02-17T13:14:43.497+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T13:14:43.569+0000] {logging_mixin.py:137} INFO - [2023-02-17T13:14:43.566+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T13:14:43.570+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T13:14:43.717+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 0.262 seconds
[2023-02-17T13:15:15.817+0000] {processor.py:153} INFO - Started process (PID=5996) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T13:15:15.819+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T13:15:15.849+0000] {logging_mixin.py:137} INFO - [2023-02-17T13:15:15.849+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T13:15:16.051+0000] {logging_mixin.py:137} INFO - [2023-02-17T13:15:16.048+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T13:15:16.063+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T13:15:16.444+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 0.809 seconds
[2023-02-17T13:15:56.150+0000] {processor.py:153} INFO - Started process (PID=6054) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T13:15:56.215+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T13:15:56.217+0000] {logging_mixin.py:137} INFO - [2023-02-17T13:15:56.217+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T13:15:56.366+0000] {logging_mixin.py:137} INFO - [2023-02-17T13:15:56.336+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T13:15:56.367+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T13:15:56.819+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 0.700 seconds
[2023-02-17T13:16:29.052+0000] {processor.py:153} INFO - Started process (PID=6100) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T13:16:29.118+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T13:16:29.121+0000] {logging_mixin.py:137} INFO - [2023-02-17T13:16:29.120+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T13:16:29.283+0000] {logging_mixin.py:137} INFO - [2023-02-17T13:16:29.281+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T13:16:29.284+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T13:16:29.564+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 0.620 seconds
[2023-02-17T13:17:05.101+0000] {processor.py:153} INFO - Started process (PID=6150) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T13:17:05.149+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T13:17:05.151+0000] {logging_mixin.py:137} INFO - [2023-02-17T13:17:05.151+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T13:17:05.253+0000] {logging_mixin.py:137} INFO - [2023-02-17T13:17:05.245+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T13:17:05.254+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T13:17:05.503+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 0.435 seconds
[2023-02-17T13:17:36.561+0000] {processor.py:153} INFO - Started process (PID=6193) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T13:17:36.564+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T13:17:36.576+0000] {logging_mixin.py:137} INFO - [2023-02-17T13:17:36.575+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T13:17:36.689+0000] {logging_mixin.py:137} INFO - [2023-02-17T13:17:36.687+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T13:17:36.726+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T13:17:37.288+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 0.844 seconds
[2023-02-17T13:18:11.087+0000] {processor.py:153} INFO - Started process (PID=6240) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T13:18:11.102+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T13:18:11.171+0000] {logging_mixin.py:137} INFO - [2023-02-17T13:18:11.136+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T13:18:11.363+0000] {logging_mixin.py:137} INFO - [2023-02-17T13:18:11.360+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T13:18:11.368+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T13:18:11.797+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 0.805 seconds
[2023-02-17T13:18:46.862+0000] {processor.py:153} INFO - Started process (PID=6296) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T13:18:46.866+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T13:18:46.878+0000] {logging_mixin.py:137} INFO - [2023-02-17T13:18:46.877+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T13:18:46.947+0000] {logging_mixin.py:137} INFO - [2023-02-17T13:18:46.941+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T13:18:46.949+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T13:18:47.053+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 0.213 seconds
[2023-02-17T13:19:21.976+0000] {processor.py:153} INFO - Started process (PID=6351) to work on /opt/airflow/dags/ingest_data.py
[2023-02-17T13:19:22.005+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/ingest_data.py for tasks to queue
[2023-02-17T13:19:22.007+0000] {logging_mixin.py:137} INFO - [2023-02-17T13:19:22.006+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/ingest_data.py
[2023-02-17T13:19:22.063+0000] {logging_mixin.py:137} INFO - [2023-02-17T13:19:22.061+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/ingest_data.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/ingest_data.py", line 10, in <module>
    from ingest_script import ingest_callable
ModuleNotFoundError: No module named 'ingest_script'
[2023-02-17T13:19:22.134+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/ingest_data.py
[2023-02-17T13:19:22.726+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/ingest_data.py took 0.784 seconds
